### ISSUE TITLE
Project Management and Team communication tools for labs

### ISSUE THREAD

**Issue by vortexing** [FredHutch/wiki#448]
### URL of existing content (a rendered, sciwiki.fredhutch.org link)
https://sciwiki.fredhutch.org/generation/labman_software/

### Specific information requests or concerns, or specific region on page of content to refresh.
I think we could benefit from tuning this page up a bit by checking in with some groups about what is available (microscoft teams??), but also what they are actively using (probably something else).  

I know one group using Basecamp (@sitapriyamoorthi ) for team communication, etc.  I'm sure there are others. Perhaps if people know more they can post here to add them and I/we can summarize them in the page?

**Comment by sitapriyamoorthi** [comment]
Yes, I think adding a page on project management would be very helpful. Would you like me to write something up?

**Comment by vortexing** [comment]
Sure!!  You can do an edit directly in that page, OR you can put text here in the comments and I can incorporate it.  Whichever works for you!

**Comment by lakikowolfe** [comment]
Hi - wondering what the status on this is? I started using Notion for project management while in Gottardo Lab and think it'd be worthwhile writing up for this

**Comment by k8hertweck** [comment]
@lakikowolfe doesn't look like there's any work happening on this right now; feel free to create a branch and start adding content/notes
